### PR TITLE
Registration ui

### PR DESCRIPTION
**Pull request by ubermensh**
Basic react registration / login workflow, integrated with backend

### CODE REVIEW

File: client/src/utils/search.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+export const searchOnliner = (query)  => {
```
Comment (AndreiPiatrou) [inline client/src/utils/search.js]: the client has to send requests to back-end and should not be aware of what data source it uses under the hood.

File: client/src/serviceWorker.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
+// This optional code is used to register a service worker.
```
Comment (AndreiPiatrou) [inline client/src/serviceWorker.js]: do we need service workers right now?

File: client/src/utils/search.js (hunk 1 above)
Comment (ubermensh) [reply client/src/utils/search.js]: Yes, I meant that. That request will go to back-end, the route there is not ready yet.

Comment (AndreiPiatrou) [reply client/src/utils/search.js]: right, so FE should not contain such BE-related specifics in the code (onliner I mean)

File: client/src/serviceWorker.js (hunk 2 above)
Comment (ubermensh) [reply client/src/serviceWorker.js]: No, it is generated by standard create-react-app. Will remove.

**Pull request merged**